LICENSE-MIT

```diff
@@ -0,0 +1,27 @@
+MIT License
+
+Copyright (c) 2017-2021 The Rust Project Developers
```

Comment:
Could you please update also this year, and remove the merge commit? Thank you! :)

Reply:
Done. Also squash?

Reply:
Yes, please! :)